### PR TITLE
Bypass fingerprint validation for skirmish/mission servers.

### PR DESCRIPTION
**Pull request by pchote**
This PR addresses the easy part of #17536. Servers created for mission/skirmish games will start immediately without attempting to validate the fingerprint.

### CODE REVIEW

**Review by Mailaender — DISMISSED** [review]
Looks good. Very sensible change.

File: OpenRA.Game/Server/Server.cs [hunk 1]
```diff
@@ -502,10 +509,10 @@ void ValidateClient(Connection newConn, string data)
...
 						delayedActions.Add(() =>
 						{
-							var notAuthenticated = Dedicated && profile == null && (Settings.RequireAuthentication || Settings.ProfileIDWhitelist.Any());
-							var blacklisted = Dedicated && profile != null && Settings.ProfileIDBlacklist.Contains(profile.ProfileID);
-							var notWhitelisted = Dedicated && Settings.ProfileIDWhitelist.Any() &&
-								(profile == null || !Settings.ProfileIDWhitelist.Contains(profile.ProfileID));
+							var notAuthenticated = Type == ServerType.Dedicated && profile == null && (Settings.RequireAuthentication || Settings.ProfileIDWhitelist.Any());
+							var blacklisted = Type == ServerType.Dedicated && profile != null && Settings.ProfileIDBlacklist.Contains(profile.ProfileID);
+							var notWhitelisted = Type == ServerType.Dedicated && Settings.ProfileIDWhitelist.Any() &&
+							                     (profile == null || !Settings.ProfileIDWhitelist.Contains(profile.ProfileID));
```
Comment (abcdefg30) [inline OpenRA.Game/Server/Server.cs]: Whitespaces.

File: OpenRA.Game/Server/Server.cs [hunk 2]
```diff
@@ -502,10 +515,10 @@ void ValidateClient(Connection newConn, string data)
...
```
Comment (abcdefg30) [inline OpenRA.Game/Server/Server.cs]: Whitespaces.

**Comment by pchote** [comment]
Fixed.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/2e3a67450ad3154e8ccdb3ea51b85e36dc77860c...80caa8745efae014ab04d7691eb7d0c87fec714e)

https://github.com/OpenRA/OpenRA/commit/16e18c1cb2deea9f802f43a21d5fff096c4bf656

**Pull request merged**